packages/react-instantsearch-dom-maps/src/Provider.js

```diff
@@ -121,13 +99,25 @@ class Provider extends Component {
         ? this.createBoundingBoxFromHits(hits)
         : currentRefinement;
 
-    return children({
-      onChange: this.onChange,
-      onIdle: this.onIdle,
-      shouldUpdate: this.shouldUpdate,
-      boundingBox,
-      boundingBoxPadding,
-    });
+    return (
+      <GeoSearchContext.Provider
+        value={{
```

Comment:
We should not use inline value here, it will re-render the `Consumer` on each render. We have [to always provide the same reference](https://reactjs.org/docs/context.html#caveats) unless the state changes.